pgjdbc/src/main/java/org/postgresql/util/PGInterval.java

```diff
@@ -98,13 +98,13 @@ private void parseISO8601Format(String value) {
       for (int i = 0; i < timeValue.length(); i++) {
         int lookAhead = lookAhead(timeValue, i, "HMS");
         if (lookAhead > 0) {
-          number = Integer.parseInt(timeValue.substring(i, lookAhead));
+          final String substring = timeValue.substring(i, lookAhead);
```

Comment:
Do we even need the substring variable at all ?

Reply:
I guess not, done.